FHICORC.Backend/FHICORC.ApplicationHost.Hangfire/Tasks/UpdateRevocationListTask.cs

```diff
@@ -51,10 +60,14 @@ public async Task UpdateRevocationList()
             var failure = false;
             DgcgRevocationBatchListRespondDto revocationBatchList = new DgcgRevocationBatchListRespondDto();
 
-            //_revocationService.DeleteExpiredBatches();
+            var api = JobStorage.Current.GetMonitoringApi();
+            var succeededJobs = api.SucceededJobs(0, int.MaxValue);
+            var lastSucceedetDate = succeededJobs.FirstOrDefault(s => s.Value.Job.Type.Name == "UpdateRevocationListTask").Value?.SucceededAt;
+           
             try
             {
-                revocationBatchList = await _dgcgService.GetRevocationBatchListAsync();
+                var modifiedSince = lastSucceedetDate != null ? lastSucceedetDate.ToString() : "2021-06-01T00:00:00Z";
```

Comment:
Can we not use DateTime instead of strings for this logic?

Reply:
No, in this logic we are tapping into some Hangfire tables to get last succeeded job

![image](https://user-images.githubusercontent.com/44182405/172307234-2fdd3a47-680c-49ac-8faa-5bb87d299460.png)

NOTE, that we have set up a task to remove this particular code. We want to save last succeeded state in a own table instead of using the hangfire tables as the hangfire tables might reset themselves after some time.

Reply:
Hangfire already returns a datetime?

So it should be no issue changing it to this:

`var modifiedSince = lastSucceedetDate != null ? lastSucceedetDate : new DateTime(2021, 6, 1, 0, 0, 0, DateTimeKind.Utc);`